### PR TITLE
WEBUI-191: fix drawer navigation by keyboard

### PR DESCRIPTION
**Pull request by semisse**

### CODE REVIEW

**Review by mnixo — APPROVED** [review]

**Review by richardsd — APPROVED** [review]

File: elements/nuxeo-app/nuxeo-menu-icon.js [hunk 1]
```diff
@@ -145,4 +153,18 @@ Polymer({
       return this.urlFor(...[name].concat(args));
     }
   },
+
+  _dispatchEvent(e) {
+    // if space or enter keys were pressed
+    if (e && e.keyCode === 13 || e.keyCode === 32) {
```
Comment (mnixo) [inline elements/nuxeo-app/nuxeo-menu-icon.js:159]: This is correct, but we're now relying purely on the precedence order of the logical operators ([which might not be taken generally as a good practice](https://softwareengineering.stackexchange.com/questions/201175/should-i-use-parentheses-in-logical-statements-even-where-not-necessary)).
If that's fine with everyone, I'm fine with it too.

**Review by mnixo — APPROVED** [review]

File: elements/nuxeo-app/nuxeo-menu-icon.js (hunk 1 above)
Comment (semisse) [reply elements/nuxeo-app/nuxeo-menu-icon.js:159]: yes you are correct, it's more robust wrapping them with parenthesis. I will do that, thanks

**Review by mnixo — APPROVED** [review]

**Review by richardsd — DISMISSED** [review]

File: elements/nuxeo-app/nuxeo-menu-icon.js [hunk 2]
```diff
@@ -145,4 +153,18 @@ Polymer({
       return this.urlFor(...[name].concat(args));
     }
   },
+
+  _dispatchEvent(e) {
```
Comment (Gabez0r) [inline elements/nuxeo-app/nuxeo-menu-icon.js:157]: Don't think we need this if this element implements `PaperItemBehavior`:
```
import { PaperItemBehavior } from '@polymer/paper-item/paper-item-behavior.js';
...
behaviors: [I18nBehavior, RoutingBehavior, PaperItemBehavior],
```

Comment (semisse) [reply elements/nuxeo-app/nuxeo-menu-icon.js]: Just tested with `PaperItemBehavior` and it seems to do the work 💪

**Review by mnixo — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]

**Pull request merged**